content/shared/v3-core-get-started/_index.md

```diff
@@ -324,10 +324,34 @@ For more information, see [diskless architecture](#diskless-architecture).
 > [!Note]
 > ##### Write requests return after WAL flush
 >
-> Because InfluxDB sends a write response after the WAL file has been flushed to the configured object store (default is every second), individual write requests might not complete quickly, but you can make many concurrent requests to achieve higher total throughput.
-> Future enhancements will include an API parameter that lets requests return without waiting for the WAL flush.
+> Because InfluxDB sends a write response after the WAL file has been flushed to the configured object store (default is every second), individual write requests might not complete quickly, but you can make many concurrent requests to achieve higher total throughput. Future enhancements will include an API parameter that lets requests return without waiting for the WAL flush.
```

Comment:
```suggestion
> By default, when you write data, InfluxDB sends a write response after the WAL file has been flushed to the Object store (default is every second). Write requests might not complete quickly, but you can make many concurrent requests to achieve higher total throughput.
For faster write responses, you can use the [`no_sync` option](#no-sync-write-option) to skip the wait for WAL persistence.
```

Reply:
`no_sync` is the "future enhancement".

Also, just FYI, our practice is to not include forecasting about future features or enhancements--we document the thing as it is, not as it might be later. However, we kept it here b/c it seemed important to the team and the early release status.